public/app/features/dashboard/containers/DashboardPage.test.tsx

```diff
@@ -322,7 +322,6 @@ describe('DashboardPage', () => {
     it('should render dashboard page toolbar and submenu', async () => {
       setup({ dashboard: getTestDashboard() });
       expect(await screen.findAllByTestId(selectors.pages.Dashboard.DashNav.navV2)).toHaveLength(1);
-      expect(screen.getAllByLabelText(selectors.pages.Dashboard.SubMenu.submenu)).toHaveLength(1);
```

Comment:
Please correct me on this, but in this case the dashboard doesn't have anything that is supposed to be displayed in the submenu? Should I add some of that data to the dashboard setup instead?

Reply:
Hey @tskarhed 👋🏾 , that's an interesting finding.

I wonder if the `dashboard setup` somehow configures something?

I know that the submenu is rendered if we have links, annotations or variables. Is it maybe the built-in annotation that is being configured?

Reply:
Yeah, the test failed, which means that the SubMenu component returned `null` previously, just having the `section` element empty. Which means that this assertion would always pass as true, unless you were in edit mode or kiosk mode, regardless of the content of the submenu.

Reply:
But based on the test, I should probably add something to the setup that makes the submenu display.

Reply:
Yes, maybe a quick change will be to add links. 🙃